cli/constants.py

```diff
@@ -1,12 +1,16 @@
 """
 Provide constants for command line interface.
 """
+from remme.models.utils.family_name import RemmeFamilyName
+
 ADDRESS_REGEXP = r'^[0-9a-f]{70}$'
 BATCH_ID_REGEXP = r'^[0-9a-f]{128}$'
+PUBLIC_KEY_REGEXP = r'^[0-9a-f]{66}$'
```

Comment:
Delete this line because is repeated in line 10.

Reply:
@anastasiia-bilova, done.